Updates to dbt Explorer (#4537)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/explore-projects.md

- Zoom in and out on the graph by mouse-scrolling.
- Grab and move the graph and the nodes.
- Right click on a node (context menu) to:
    - Refocus on the node, including its parent and child nodes
    - Refocus on the node and its children only
    - Refocus on the node and it parents only
    - View the node's [resource details](#view-resource-details) page

- Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details.
    - Click the Share icon in the side panel to copy the graph’s link to your clipboard.
    - Click the View Resource icon in the side panel to [view the resource details](#view-resource-details). 
- [Search and select specific resources](#search-resources) or a subset of the DAG using [selectors](/reference/node-selection/methods) and [graph operators](/reference/node-selection/graph-operators). This can help you narrow the focus on the resources that interest you. For example:

>     - `+[RESOURCE_NAME]` &mdash; Displays all parent nodes of the resource

Reviewer: non blocking sugg - i like the example and i would suggest sharing a real-life example that users new to explorer will immediately understand, as opposed to only displaying a generic 'resource_type:model [resource_name]` example.

Reviewer: similar to how you display an example in 'Search with selectors'

### Search with keywords
When searching with keywords, dbt Explorer searches through your resource metadata (such as resource type, resource name, column name, source name, tags, schema, database, version, alias/identifier, and package name) and returns any matches.

> With keyword search, there is a side panel (to the right of the main section) that you can use to additionally filter the search results by resource type. You can filter by resource tags for all resources or model access levels within the **Models** option. As an example, when searching for “sale”, the results will include all resources with the keyword “sale” in their metadata. And, choosing **Models** and **Sources** in the side panel will filter those search results to contain only models or sources.   

Reviewer: sugg bullets or shortening so it's easier to read/scan

```suggestion
- Keyword search features a side panel (to the right of the main section) to filter search results by resource type. 
- Use this panel to select specific resource tags or model access levels under the **Models** option. 
  - For example, a search for "sale" returns results that include all resources with the keyword "sale" in their metadata. Filtering by **Models** and **Sources** refines these results to only include models or sources.
```

> For a search on an exact column name, the results will include all relational nodes with that specific column in their schemas. If there is a match, the search result will include a notice indicating that the resource contains the specified column.

Reviewer: ```suggestion
- When searching for an exact column name, the results show all relational nodes containing that column in their schemas. If there's a match, a notice in the search result indicates the resource contains the specified column.
```
